website/source/docs/drivers/docker.html.md

```diff
@@ -368,6 +368,9 @@ The `docker` driver supports the following configuration in the job spec.  Only
 * `readonly_rootfs` - (Optional) `true` or `false` (default). Mount
   the container's filesystem as read only.
 
+* `pids_limit` - (Optional) An integer value that specifies the pid limit for
```

Comment:
should this be just `pid_limit` (singular vs plural)

Reply:
`docker run` calls it `pids-limit` https://docs.docker.com/engine/reference/commandline/run/ so would prefer to keep it the same for consistency